lib/pbench/agent/tool_meister.py

```diff
@@ -122,21 +125,22 @@ def start(self):
                 + "/bindings/common"
             )
 
+            script_path = "/samples/scripts/dcgm_prometheus.py"
```

Comment:
No reason not to prepend `self.install_path` here, since it's added in all three uses of the subpath...

Reply:
I took the point of the comment to be that you could say
```
   script_path = self.install_path + "/samples/scripts/dcgm_prometheus.py"
```
and just use `script_path` below. 

And maybe use `pathlib` for path component manipulations...